iaen.edu.ec,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
iainbengkulu.ac.id,"HTTPSConnectionPool(host='iainbengkulu.ac.id', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='iainbengkulu.ac.id', port=443): Read timed out. (read timeout=2)"
iam.ma,"HTTPSConnectionPool(host='iam.ma', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.iam.ma', port=443): Max retries exceeded with url: /Pages/index.aspx (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
ias.co.il,"HTTPSConnectionPool(host='ias.co.il', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='ias.co.il', port=80): Read timed out. (read timeout=2)"
ibict.br,"HTTPSConnectionPool(host='ibict.br', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='ibict.br', port=80): Read timed out. (read timeout=2)"
icbc.com.ar,"HTTPSConnectionPool(host='icbc.com.ar', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))","HTTPSConnectionPool(host='icbc.com.ar', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))"
icbf.gov.co,"HTTPSConnectionPool(host='icbf.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='icbf.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
icm.edu.pl,"HTTPSConnectionPool(host='icm.edu.pl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='icm.edu.pl', port=443): Read timed out. (read timeout=2)"
icv.hr,"HTTPSConnectionPool(host='www.icv.hr', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='icv.hr', port=80): Read timed out. (read timeout=2)"
idi.org.il,"HTTPSConnectionPool(host='idi.org.il', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f3ece7bc400>, 'Connection to idi.org.il timed out. (connect timeout=2)'))","HTTPConnectionPool(host='idi.org.il', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3eceb2dfd0>: Failed to establish a new connection: [Errno 113] No route to host'))"
ieatishootipost.sg,"HTTPConnectionPool(host='ieatishootipost.sg', port=80): Read timed out. (read timeout=2)","HTTPConnectionPool(host='ieatishootipost.sg', port=80): Read timed out. (read timeout=2)"
iedu.sk,"HTTPSConnectionPool(host='iedu.sk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))","HTTPSConnectionPool(host='iedu.sk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))"
iefp.pt,"HTTPSConnectionPool(host='iefp.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3ecdc15b80>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='iefp.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ece5e9a30>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
ieftine.eu,Exceeded 30 redirects.,Exceeded 30 redirects.
iett.istanbul,"HTTPSConnectionPool(host='iett.istanbul', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='iett.istanbul', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
iform.dk,"HTTPSConnectionPool(host='iform.dk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='iform.dk', port=443): Read timed out. (read timeout=2)"
iform.nu,"HTTPSConnectionPool(host='iform.nu', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='iform.nu', port=443): Read timed out. (read timeout=2)"
iform.se,"HTTPSConnectionPool(host='iform.se', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='iform.se', port=443): Read timed out. (read timeout=2)"
igenyesferfi.hu,"HTTPSConnectionPool(host='igenyesferfi.hu', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='igenyesferfi.hu', port=80): Read timed out. (read timeout=2)"
igm.org.il,"HTTPSConnectionPool(host='igm.org.il', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='igm.org.il', port=80): Read timed out. (read timeout=2)"
igrackeshop.hr,"HTTPSConnectionPool(host='igrackeshop.hr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'igrackeshop.hr' doesn't match either of 'jatekshop.eu', 'www.jatekshop.eu', 'www.jucariishop.ro', 'www.igraceshop.si', 'www.hrackyshop.sk', 'www.igrackeshop.hr'"")))","HTTPSConnectionPool(host='www.igrackeshop.hr', port=443): Read timed out. (read timeout=2)"
iif.hu,"HTTPSConnectionPool(host='iif.hu', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3ed09625e0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='iif.hu', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ece4bde20>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
ijf.hr,"HTTPSConnectionPool(host='ijf.hr', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f3eccb991c0>, 'Connection to ijf.hr timed out. (connect timeout=2)'))","HTTPConnectionPool(host='ijf.hr', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f3ecf0e14f0>, 'Connection to ijf.hr timed out. (connect timeout=2)'))"
ijunoon.com,"HTTPSConnectionPool(host='ijunoon.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='ijunoon.com', port=443): Read timed out. (read timeout=2)"
ilmangione.it,"HTTPSConnectionPool(host='ilmangione.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.ilmangione.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
ilmusosial.id,"HTTPSConnectionPool(host='ilmusosial.id', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3ed073ad90>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='ilmusosial.id', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ed14480a0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
imamu.edu.sa,"HTTPSConnectionPool(host='imamu.edu.sa', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='imamu.edu.sa', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
imaret.com.tr,"HTTPSConnectionPool(host='imaret.com.tr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='imaret.com.tr', port=443): Read timed out. (read timeout=2)"
imf.org,"HTTPSConnectionPool(host='www.imf.org', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.imf.org', port=443): Read timed out. (read timeout=2)"
imge.com.tr,"HTTPSConnectionPool(host='www.imge.com.tr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.imge.com.tr', port=443): Read timed out. (read timeout=2)"
imged.pl,"HTTPSConnectionPool(host='imged.pl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3ecee90f40>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='imged.pl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ecee38760>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
imist.ma,"HTTPSConnectionPool(host='imist.ma', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3ed0757d90>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='imist.ma', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ed14e2100>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
immoproxio.be,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
imprentanacional.go.cr,"HTTPSConnectionPool(host='imprentanacional.go.cr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3ed0e2db80>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='imprentanacional.go.cr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ed037d3a0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
impreseita.com,"HTTPSConnectionPool(host='impreseita.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3ed1296520>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))","HTTPConnectionPool(host='impreseita.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ecb4a8400>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))"
imt-ip.pt,"HTTPSConnectionPool(host='imt-ip.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3ecf364b80>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='imt-ip.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ed060f850>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
imtranslator.net,"HTTPSConnectionPool(host='imtranslator.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='imtranslator.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
imunita.online,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
in.gr,"HTTPSConnectionPool(host='www.in.gr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.in.gr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))"
inclusion.gob.ec,"HTTPSConnectionPool(host='inclusion.gob.ec', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'inclusion.gob.ec' doesn't match either of 'www.agricultura.gob.ec', 'agricultura.gob.ec'"")))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
indap.gob.cl,"('Connection aborted.', BadStatusLine('\n'))","HTTPConnectionPool(host='www.indap.gob.cl', port=80): Read timed out. (read timeout=2)"
indexlivingmall.com,"HTTPSConnectionPool(host='indexlivingmall.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f3ec9d76a90>, 'Connection to indexlivingmall.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='indexlivingmall.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f3ec9ccf910>, 'Connection to indexlivingmall.com timed out. (connect timeout=2)'))"
indonesia.go.id,"HTTPSConnectionPool(host='indonesia.go.id', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.indonesia.go.id', port=443): Read timed out. (read timeout=2)"
ine.gob.gt,"HTTPSConnectionPool(host='ine.gob.gt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'ine.gob.gt' doesn't match 'www.ine.gob.gt'"")))","HTTPSConnectionPool(host='www.ine.gob.gt', port=443): Read timed out. (read timeout=2)"
ine.gub.uy,"HTTPSConnectionPool(host='ine.gub.uy', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='ine.gub.uy', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
inefop.org.uy,"HTTPSConnectionPool(host='inefop.org.uy', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3ecd221100>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='inefop.org.uy', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3eca4c74f0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
infarmed.pt,"HTTPSConnectionPool(host='infarmed.pt', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f3ecec60940>, 'Connection to infarmed.pt timed out. (connect timeout=2)'))","HTTPConnectionPool(host='infarmed.pt', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f3ed114f7c0>, 'Connection to infarmed.pt timed out. (connect timeout=2)'))"
infinitydesign.in.th,"HTTPSConnectionPool(host='www.infinitydesign.in.th', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.infinitydesign.in.th', port=443): Read timed out. (read timeout=2)"
infocenters.co.il,"HTTPSConnectionPool(host='infocenters.co.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='infocenters.co.il', port=443): Read timed out. (read timeout=2)"
infogate.cl,"HTTPSConnectionPool(host='www.infogate.cl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.infogate.cl', port=443): Read timed out. (read timeout=2)"
infojuht.ee,"HTTPSConnectionPool(host='infojuht.ee', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='infojuht.ee', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
infonegocios.info,"HTTPSConnectionPool(host='infonegocios.info', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='infonegocios.info', port=443): Read timed out. (read timeout=2)"
infoprision.com,"HTTPSConnectionPool(host='infoprision.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'infoprision.com' doesn't match 'www.infoprision.com'"")))","HTTPSConnectionPool(host='infoprision.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'infoprision.com' doesn't match 'www.infoprision.com'"")))"
infoproff.com,"HTTPSConnectionPool(host='infoproff.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.infoproff.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
infopublik.id,"HTTPSConnectionPool(host='infopublik.id', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='infopublik.id', port=443): Read timed out. (read timeout=2)"
informa.es,"HTTPSConnectionPool(host='www.informa.es', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.informa.es', port=443): Read timed out. (read timeout=2)"
information.dk,"HTTPSConnectionPool(host='information.dk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.information.dk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))"
informiran.si,"HTTPSConnectionPool(host='informiran.si', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='informiran.si', port=80): Read timed out. (read timeout=2)"
ing.be,"HTTPSConnectionPool(host='ing.be', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='ing.be', port=80): Read timed out. (read timeout=2)"
inia.cl,"HTTPSConnectionPool(host='www.inia.cl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.inia.cl', port=443): Read timed out. (read timeout=2)"
inn.no,"HTTPSConnectionPool(host='inn.no', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3ec8b8d6d0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPSConnectionPool(host='inn.no', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3eca44a850>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
ins-cr.com,"HTTPSConnectionPool(host='ins-cr.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3ed0d91670>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='ins-cr.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ec8bfe250>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
insse.ro,"HTTPSConnectionPool(host='insse.ro', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3eca57edf0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='insse.ro', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ed0984460>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
intercars.com.pl,"HTTPSConnectionPool(host='intercars.com.pl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='intercars.com.pl', port=443): Read timed out. (read timeout=2)"
intermarche.pt,"HTTPSConnectionPool(host='www.intermarche.pt', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.intermarche.pt', port=443): Read timed out. (read timeout=2)"
internimedicina.cz,"HTTPSConnectionPool(host='internimedicina.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))","HTTPSConnectionPool(host='internimedicina.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))"
intima.hu,Exceeded 30 redirects.,Exceeded 30 redirects.
investireoggi.it,"HTTPSConnectionPool(host='www.investireoggi.it', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.investireoggi.it', port=443): Read timed out. (read timeout=2)"
invima.gov.co,"HTTPSConnectionPool(host='invima.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='invima.gov.co', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
io.ua,"HTTPSConnectionPool(host='io.ua', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3ecaed3a60>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='io.ua', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ec49215b0>: Failed to establish a new connection: [Errno 111] Connection refused'))"
ioc.ee,"HTTPSConnectionPool(host='ioc.ee', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.ioc.ee', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
ipapercms.dk,"HTTPSConnectionPool(host='ipapercms.dk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3eceb2dc70>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='ipapercms.dk', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ec9b6dee0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
ipleaders.in,"HTTPSConnectionPool(host='ipleaders.in', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='ipleaders.in', port=443): Read timed out. (read timeout=2)"
ipoor.org,"HTTPSConnectionPool(host='ipoor.org', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3ecc871d00>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ipoor.org', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ec4a2fd00>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ipsantarem.pt,"HTTPSConnectionPool(host='ipsantarem.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3ec9b84820>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='ipsantarem.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ed001ec70>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
ipst.ac.th,"HTTPSConnectionPool(host='ipst.ac.th', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f3ecb0b87f0>, 'Connection to ipst.ac.th timed out. (connect timeout=2)'))","HTTPConnectionPool(host='ipst.ac.th', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f3ed13f55b0>, 'Connection to ipst.ac.th timed out. (connect timeout=2)'))"
ipvc.pt,"HTTPSConnectionPool(host='ipvc.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3ecf4965e0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='ipvc.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3ed03375b0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
ir.lv,"HTTPSConnectionPool(host='ir.lv', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='ir.lv', port=443): Read timed out. (read timeout=2)"
ird.gov.lk,"HTTPSConnectionPool(host='ird.gov.lk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3ec7e741f0>: Failed to establish a new connection: [Errno 113] No route to host'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
